x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/common/text/TextTemplateTests.java

```diff
@@ -113,6 +114,13 @@ public String execute() {
         assertThat(engine.render(template, model), is("rendered_text"));
     }
 
+    public void testDontInvokeScriptServiceOnNonMustacheText() {
+        String input = rarely() ? "}}{{" : "this is my text";
```

Comment:
If we are going to have this special case, why can't we test it every time, instead of relying on randomization?